Khan/perseus
# Check for in progress releases before trying to do snapshots

## Summary:
This PR modifies our node-ci workflow so that we don't bother trying to do snapshots if we see a release workflow in progress.

Issue: FEI-6062

## Test plan:
To test this out, we will need to:

1. Have a Version Packages release ready to merge (DO NOT MERGE YET)
2. Have a PR ready that will trigger a snapshot (a simple change in an existing package, like a comment - we're not actually landing the PR)
3. Merge that release
4. Immediately `git pull && git merge origin/main && git push` on the PR
5. Watch the workflows. The release workflow should succeed and publish the packages. The snapshot workflow should fail with a message about a release in progress, and it should update the PR comment saying the snapshot was not created.
